vendor/github.com/jacobsa/gcloud/gcs/bucket.go

```diff
@@ -350,13 +352,25 @@ func (b *bucket) DeleteObject(
 }
 
 func newBucket(
+	ctx context.Context,
 	client *http.Client,
 	url *url.URL,
 	userAgent string,
 	name string,
 	billingProject string) Bucket {
+
+	// Creating client through GSCL for the sclClient parameter of bucket.
+	var sclClient *storage.Client = nil
+	sclClient, err := storage.NewClient(ctx)
+	if err != nil {
+		err = fmt.Errorf("Error in creating the client through GSCL: %v", err)
```

Comment:
what exactly are you doing here. formatting the error message but neither we are returning the error nor printing it, what does it mean?

Reply:
Yeah I'll change it to print the error.